Updated Chapter 2 (#233)
starknet-edu/starknetbook: src/ch02-02-compile-deploy-interact.md

the Account Descriptor is a JSON file detailing the wallet’s address and
public key.

> For an account to be used as a signer it must be deployed to the appropriate network (Starknet Goerli or mainnet) and funded.

Reviewer: ```suggestion
In order for an account to be used as a signer it must be deployed to the appropriate network, Starknet Goerli or mainnet, and funded.
```

Steps for deploying and funding an account: 

1. Install Braavos or Argent X browser extensions.

> 2. Fund your account with ETH. Click on "Deposit" on Braavos or "Add funds" on Argent X, then select 

Reviewer: ```suggestion
2. Fund your account with ETH. Click  **Deposit** in Braavos or **Add funds** in Argent X, then select 
```

Reviewer: Avoid quotation marks. They create an ironic tone. For GUI items, use bold.

>     "Starknet token faucet" to get testnet tokens. You can also use [Starknet Goerli

Reviewer: ```suggestion
    **Starknet token faucet** to get testnet tokens. You can also use the [Starknet Goerli
```

    Faucet](https://faucet.goerli.starknet.io/).

> 3. For Braavos, on the account page click on "set up your account on-chain" to deploy your account. 

Reviewer: ```suggestion
3. For Braavos, on the account page, deploy your account by clicking **set up your account on-chain**. 
```

Reviewer: Put the action that you're doing before the GUI interaction to give motivation for the GUI action.

    For Argent X, fund your account using the faucet (for testnet) or fund using Ether, then make an 

>     outgoing transaction from the account (you can just send some tokens to account2 in your wallet).

Reviewer: ```suggestion
    outgoing transaction from the account. For example, you can just send some tokens to account2 in your wallet.
```
A general technical writing guideline for parentheses is to avoid them. Usually the parenthetical phrase falls into one of these:
* Important enough for its own sentence.
* Not important, so shouldn't be there.
I use parenthese for labeling, like you did in the first part of the sentence: `(for testnet)`.

>     This will automatically deploy your account.

Reviewer: I try to add a noun after the word "This" for clarity. Also, use the present tense instead of the future, unless you can't avoid the future tense.
```suggestion
    This transaction automatically deploys your account.
```

Author: Thank you very much, the changes have been effected.

the Account Descriptor is a JSON file detailing the wallet’s address and
public key.

In order for an account to be used as a signer it must be deployed to the appropriate network, 
Starknet Goerli or mainnet, and funded.

Steps for deploying and funding an account: 

1. Install Braavos or Argent X browser extensions.

2. Fund your account with ETH. Click on **Deposit** on Braavos or **Add funds** on Argent X, 
  then select **Starknet token faucet** to get testnet tokens. 
  You can also use the [Starknet Goerli Faucet](https://faucet.goerli.starknet.io/).

3. For Braavos, on the account page, deploy your account by clicking on **set up your account on-chain**. 
    For Argent X, fund your account using the faucet (for testnet) or fund using Ether, 
    then make an outgoing transaction from the account. For example, you can just send 

>     some tokens to account2 in your wallet. This transaction automatically deploys your account.

Reviewer: ```suggestion
    some tokens to account2 from your wallet. This transaction automatically deploys your account.
```
